agents/langgraph/examples/guardrailed_agent/Makefile

```
@@ -0,0 +1,212 @@
+	      echo "ERROR: Ollama failed to start within 30 seconds." && \
+	      kill $$OLLAMA_PID 2>/dev/null; \
+	      exit 1; \
+	    fi; \
+	    sleep 1; \
+	  done && \
+	  echo "==> Pulling $(MODEL)..." && \
+	  ollama pull $(MODEL) && \
+	  echo "" && \
+	  echo "Done. Run 'make guardrails-server' to start NeMo Guardrails."
+
+guardrails-server: ## Generate config from .env and start NeMo Guardrails proxy
```

checkmake (0.3.2)

[warning] Target body for "guardrails-server" exceeds allowed length of 5 lines (14).

(maxbodylength)